lib/web/autoupdate_common.go

```diff
@@ -0,0 +1,220 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package web
+
+import (
+	"context"
+	"strings"
+
+	"github.com/gravitational/trace"
```

Comment:
nit: why isolate this import?

Reply:
Imports are ordered and enforced by `gci` with the following configuration:

https://github.com/gravitational/teleport/blob/e5db27387fe81b2cd09cc8ebb06a2bdcee484d8a/.golangci.yml#L225-L232

This import falls into the `default` section.